.github/workflows/eslint.yml

```
@@ -0,0 +1,43 @@
+name: "ESLint"
+
+on: [push, pull_request]
+
+jobs:
+  frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16.x"
+
+      - name: "/frontend: Install Node.js dependencies"
+        working-directory: frontend
+        run: |
+          npm install -g yarn
+          yarn install --frozen-lockfile
+
+      - name: "/frontend: Linting"
+        working-directory: frontend
+        run: yarn run lint
+
+  api:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16.x"
+
+      - name: "/api: Install dependencies"
+        run: |
+          cd api
```

For consistency with the rest of the workflow:

```suggestion
        working-directory: api
        run: |
```

---

Nice catch! Tested it and seems to work as expected :+1: